packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -20,11 +20,20 @@ import ToggleAdvancedSettingsButton from './controls/ToggleAdvancedSettingsButto
 import shouldShowMissingPasswordWarning from '@joplin/lib/components/shared/config/shouldShowMissingPasswordWarning';
 import MacOSMissingPasswordHelpLink from './controls/MissingPasswordHelpLink';
 const { KeymapConfigScreen } = require('../KeymapConfig/KeymapConfigScreen');
+import FontSearch from './FontSearch';
 
 const settingKeyToControl: any = {
 	'plugins.states': control_PluginsStates,
 };
 
+interface FontData {
+	family: string;
+	fullName: string;
+	postscriptName: string;
+	style: string;
+	blob: ()=> Promise<Blob>;
```

Comment:
In my comment I mentioned defining the properties that you need and you don't need most of these. I think you only use .family?

When a type is not built-in we just define what we need, otherwise it's too much noise especially for the DOM which can have hundreds of properties per object.